fcc-test-suite-template-master/index.html

```diff
@@ -0,0 +1,75 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.0-beta2/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-BmbxuPwQa2lc/FVzBcNJ7UAyJxM6wuqIj61tLrc4wSX0szH/Ev+nYRRuWlolflfl" crossorigin="anonymous">
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+
+  <body>
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+      <div>
+        <div class="nav d-flex flex-row">
+          <nav class="navbar navbar-expand-lg navbar-dark bg-dark fixed-top">
+            <a class="navbar-brand" href="#">Navbar</a>
+            <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarSupportedContent" aria-controls="navbarSupportedContent" aria-expanded="false" aria-label="Toggle navigation">
+              <span class="navbar-toggler-icon"></span>
+            </button>
+            <div class="collapse navbar-collapse" id="navbarSupportedContent">
+              <ul class="navbar-nav mr-auto">
+                <li class="nav-item active">
+                  <a class="nav-link" href="#">Home <span class="sr-only">(current)</span></a>
+                </li>
+                <li class="nav-item">
+                  <a class="nav-link" target="_blank" href="https://github.com/BethrandChukwulobe">Link Beth</a>
+                </li>
+                <li class="nav-item">
+                  <a class="nav-link" target="_blank" href="https://github.com/jchernandez87">Link Juan</a>
+                </li>
+              </ul>
+            </div>
+          </nav>
+        </div>
+        <div class="welcome d-flex flex-column align-items-center justify-content-center">
+          <h1>Hello World!</h1>
+          <h3>We are software developers</h3>
+        </div>
+        <div class="projects d-flex flex-row justify-content-around mt-5 mb-5">
+          <div class="card" style="width: 18rem;">
+            <img class="card-img-top" src="./assets/img/project1.jpg" alt="Card image cap">
+            <div class="card-body">
+              <h5 class="card-title">Project 1</h5>
+              <p class="card-text">Some quick example text to build on the card title and make up the bulk of the card's content.</p>
+              <a href="https://csanpedroclaver.edu.co/" target="_blank" class="btn btn-primary">Click me!</a>
```

Comment:
Link 'rel' attribute should include 'noreferrer' or no opener try using href="#"

Reply:
Kindly apply my first inline comment to reconstruct this too.